### PR TITLE
SPCTR-1966 New Block - Style settings added for modal block

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
This task will include styling options for the modal popup triggers and their content. It will include the following settings - 
Modal Popup - overlay, box-shadow, padding, etc.
Modal trigger
Modal close button

Also, it should work correctly at the front.

### Types of changes

New

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -175,6 +175,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Improvement: Post Block - Woo Commerce Products (CPT) not getting in post Type.
 * Improvement: Block Transform - Transform from core Paragraph/List to Spectra Heading/Blockquotes.
 * Improvement: Post Blocks - Optimize the loading of Post Terms.
+* Improvement: Table of content - Added overall block alignment option.
```
Comment (bsf-zanev) [inline README.md:178]: @AbhijitNage123 - Is this PR ahead of its base branch final-modal?

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
The updated code is as per standards, but this branch is ahead of its base branch.

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Updated

**Review by bsf-zanev — APPROVED** [review]
Verified Changes.

**Review by AbhijitNage123 — APPROVED** [review]

**Pull request merged**